fmriprep/cli/run.py

```diff
@@ -214,11 +212,17 @@ def main():
         from fmriprep.reports.core import generate_reports
 
         # Generate reports phase
+        session_list = (
+            config.execution.bids_filters["bold"]["session"]
+            if config.execution.bids_filters
+            else None
+        )
```

Comment:
keys "bold" and/or "session" are not guaranteed to exist at this point. If they don't, you'll get KeyError.

```suggestion
        session_list = config.execution.bids_filters.get("bold", {}).get("session", None)
```